### DISCUSSION TITLE
How can I get the properties header to show so I can collapse it?

### DISCUSSION THREAD

**Discussion by Eneswar** [chrisgrieser/shimmering-focus#293] [Q&A] [answered]
What title says, thank you.

**Comment by chrisgrieser** [comment]
Could you post a screenshot which property headers you mean? Iirc, the theme does not hide the property folding buttons.

**Reply by Eneswar** [reply]
Certainly. I will show it on the sandbox vault (the properties button on the top): 
![image](https://github.com/chrisgrieser/shimmering-focus/assets/22943781/d15d748e-bd17-4e66-80b1-df0b3a5588f0)

And this is how it is on the theme where I cant seem to collapse it:
![image](https://github.com/chrisgrieser/shimmering-focus/assets/22943781/15a1df37-3b4f-483e-90f9-ea90a52b1903)

**Reply by chrisgrieser** [reply, marked as answer]
That here cannot be shown currently. You can nonetheless fold the properties  

Enable "fold headings" and a small triangle will appear to the left of the properties which you can use for folding.

**Reply by Eneswar** [reply]
Thank you!
